src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1239,7 +1239,7 @@ private void PrintTargetNamePerMessage(BuildMessageEventArgs e, bool lightenText
                 string targetName = string.Empty;
 
                 // Does the context (Project, Node, Context, Target, NOT task) of the previous event match the current message
-                bool contextAreEqual = s_compareContextNodeIdTargetId.Equals(currentBuildEventContext, _lastDisplayedBuildEventContext == null ? null : _lastDisplayedBuildEventContext);
+                bool contextAreEqual = s_compareContextNodeIdTargetId.Equals(currentBuildEventContext, _lastDisplayedBuildEventContext ?? null);
```

Comment:
Is this even worth a null check? Even if not, I'd rather not make the change in this PR so we keep this one focused.

Reply:
There was a refactor suggestion to remove it. So I did. I think it comes under a simplification of null checks.